ci/nightly/pipeline.template.yml

```
@@ -1062,6 +1062,17 @@ steps:
     agents:
       queue: linux-x86_64
 
+  - id: legacy-upgrade
+    label: Legacy upgrade tests (git-based)
```

git-based might not be immediately clear to everyone. Maybe 
```suggestion
    label: Legacy upgrade tests (last version from git)
```